docs/upgrade/UPGRADE-v8.0.0.md

```diff
@@ -28,6 +28,7 @@ There you can find links to upgrade notes for other versions too.
 
     In case of extending one of these classes, you should add an `EntityNameResolver` to a constructor and use it in a `create()` method to resolve correct class to return.
 - run `php phing standards-fix` so all nullable values will be now defined using nullability (?) symbol ([#1010](https://github.com/shopsys/shopsys/pull/1010))
+- update PHP version in your `composer.json` in `require` and `config` section to `7.2` ([#1066](https://github.com/shopsys/shopsys/pull/1066))
```

Comment:
I'm wondering, do you really need to do this? Maybe for other developers in your project... 
I'm thinking about other things and I would like to somehow emphasizes the dropped support for 7.1. 
But I know we already have upgrade instructions to update to PHP 7.3 (but it was optional/low priority back then).

In my opinion, another sentence about dropping support can't hurt and may help someone, what do you think?

Reply:
I have added info about end of support in Shopsys Framework